devops/docker/rhel-7-amd64/Dockerfile

```diff
@@ -1,11 +1,13 @@
 # ubi7 images are missing RPM devtools, copy from centos7
 FROM centos:centos7 as centos
 RUN rm /etc/yum.repos.d/*.repo
-COPY centos.repo /etc/yum.repos.d/centos.repo
+# Docker context is at the root of the project
```

Comment:
Why do we need to repeat this same comment as many times? As this comment really does not seem to bring anything useful to the reader, can we simply remove it from all places it is appearing?